noxfile.py

```diff
@@ -24,7 +24,6 @@ def format(session):
     session.install("-r", "tests/requirements.txt")
     session.run("isort", ".")
     session.run("black", ".")
-    session.log("To get sure both formatters are happy, run 'nox -s lint'.")

```

Comment:
I removed all those warnings about "isort getting mad with black, and black getting mad with isort" from our Nox sessions.